### ISSUE TITLE
BBNP paper data

### ISSUE THREAD

**Issue by efwaring** [schwilklab/BBNP20102011Data#2]
Does anything appear to be missing before I push this to the SchwilkLab page permanently?

**Comment by dschwilk** [comment]
Hi Lizz,

Some suggestions:
1. Add outline and links to individual files from the README.
2. I don't understand the metadata format. It looks like plain text and you have followed some specific protocol?  Is that machine readable?  At least put info in the README on what format your metadata is in.  Eventually, you will need to get it in something standard for uploading to a data share site.  But for github, something machine readable we can script into another form is ok. One option, use the system we are using in the schwilk lab for documenting variable types in csv files: https://github.com/schwilklab/protocols/blob/master/data-management.md. 
3. Why is transect data in two files?
4. R code should probably be broken up tino more manageable scripts. That is one long file.  EG:   a) reading data and data shaping, b) figures, c) stats.  Or at least two files with b and c combined. 
5. I'm pretty sure leaf area was down with scanner for all data we used, right?, so metadata is wrong about method?

**Comment by efwaring** [comment]
I have fixed the metadata format.  The data collection information is in a .md file while the information on variables in .csv files is separate .csv files.  The transect data are in two files because that is the rawest form we have them in and that is how I analyzed them. I will break up the R code next.  But the metadata.datacollection.md has been updated on how leaf area was collected. It was Lamina.
